client/js/options.js

```diff
@@ -91,6 +93,11 @@ settings.on("change", "input, select, textarea", function() {
 			// otherwise, users get notifications for everything
 			return h !== "";
 		});
+		// Construct regex with wordboundary for every highlight item
+		var highlightsTokens = options.highlights.map(function(h) {
+			return escapeRegExp(h);
+		});
+		options.highlightsRE = new RegExp("\\b(?:" + highlightsTokens.join("|") + ")\\b", "i");
```

Comment:
What will this do if `highlightsTokens` is empty (e.g. when clearing out highlights in settings)